e2e/features/step_definitions/bulk-scanning.steps.ts

```diff
@@ -40,7 +40,7 @@ async function checkDataItems() {
     }
 }
 
-Given(/^I have a bulk-scanned document with (?:all fields)$/, {timeout: 240 * 1000}, async function () {
+Given(/^I have a bulk-scanned document with (?:all fields)$/, {timeout: 600 * 1000}, async function () {
```

Comment:
Do you need the timeout thing here ? If you remove `{timeout: 600 * 1000}` would the default be applied ?

Reply:
yes it would, but unfortunately AAT response time is incredibly slow so the default timeout isn't sufficient when you first land on the homepage (the start point of this step) and times out. Every other step should complete well within 60seconds, so this is the only one set to 5mins to avoid slow feedback